torchvision/models/detection/transform.py

```diff
@@ -222,8 +237,8 @@ def batch_images(self, images: List[Tensor], size_divisible: int = 32) -> Tensor
 
     def postprocess(self,
                     result: List[Dict[str, Tensor]],
-                    image_shapes: List[Tuple[int, int]],
```

Comment:
Here is the problem, We have colliding types.
We are Defining this as `Tuple[int, int]` here but down in `resize_boxes` we use List[int]
I'm not sure whether we should keep `List[int]` or `Tuple[int, int]` we would need to modify the code accordingly.

Currently due to this, we have CI failure.

 @pmeier and @datumbox

Reply:
Thought the convention seems to be using `List[List[int]]` everywhere

Reply:
> Thought the convention seems to be using `List[List[int]]` everywhere

If that is the convention, let's also roll with it here. Although I would prefer `Tuple[int, int]`, because an image size is always of length 2 and not arbitrary.

Reply:
Ok so I will refactor to `Tuple[int, int]`. This might be bc breaking as it changes the old annotation of `List[int]` and users would get error if they were not passing List of length 2.

Reply:
Why would they get an error? IF we only change the annotations, the runtime behavior should not change.